### PR TITLE
Update REST, API-doc for review

### PR DESCRIPTION
**Pull request by Runi-VN**
- Added CRUD for Cities as @Maltemagnussen implemented earlier.
- Added examples for PersonDTO's
- Minor fixes

Had a lot of trouble with caching and merge conflicts so I spent quite some time on that today.

I plan to continue on this pull before the review in the morning.

### CODE REVIEW

File: src/main/java/dto/PersonDTO_IN.java [hunk 1]
```diff
@@ -11,19 +12,19 @@
...
+    @Schema(description = "Email", required = true, example = "Johnny@Reimar.dk")
     private String email;
-    @Schema(example = "None available")
+    @Schema(example = "{\"name\":\"Lacrosse\",\"description\":\"Rich people game\"}")
     private List<HobbyDTO_IN> hobbies = new ArrayList();
-    @Schema(example = "None available")
+    @Schema(example = "{\"number\":\"12345678\",\"description\":\"work\"}")
     private List<PhoneDTO_IN> phones = new ArrayList();
-    @Schema(name = "Address", example = "None available")
+    @Schema(example = "[{\"street\":\"Vejnavn\",\"additionalInfo\":\"Nr. 12, 1tv\",\"cityInfo\":{\"zipCode\":\"2800\",\"city\":\"Lyngby\"}}]")
```
Comment (MalteMagnussen) [inline src/main/java/dto/PersonDTO_IN.java]: >Manually doing JSON
>:kms:

File: src/main/java/dto/PersonDTO_OUT.java [hunk 2]
```diff
@@ -11,19 +11,19 @@
 @Schema(name = "Person")
 public class PersonDTO_OUT {
 
-    @Schema(name = "ID", example = "1")
+    @Schema(description = "Unique Identifier", required = true, example = "1")
```
Comment (HrBjarup) [inline src/main/java/dto/PersonDTO_OUT.java]: I don't think the DTO_OUTs should have the required = true attribute. It doesn't make sense to say that something is required when it's not related to something the user is responsible for (that's what Lars said in the first video about API DOC)

Comment (Runi-VN) [reply src/main/java/dto/PersonDTO_OUT.java]: merge conflict error. will fix.

File: src/main/java/dto/PersonDTO_IN.java (hunk 1 above)
Comment (Runi-VN) [reply src/main/java/dto/PersonDTO_IN.java]: tell me about it.

I even had it prettyfied but it does **NOT** like that

**Comment by Runi-VN** [comment]
After my next push you can now use the API doc to create an user with a hobby using the respective endpoint.  
![]

However, do notice that the DTO is shown as *all* the data, when in reality we only need some of it.  
As shown in the GIF, you must delete some of the data before sending. ("must" & "must")

**Comment by Runi-VN** [comment]
Going to stop for now and would like new approvals and a merge.

Issues:
- The DTO error as described above
- I think that we should extend our addPerson-method to account for all possible fields. Currently we have addPerson and addPersonWithHobby. That is too much.

**You can now use the API to perform actions on both localhost and the deployment**  

### Given the following:
- The REST endpoint and its facade methods must work (not all are implemented)
- The JSON input in request body must be correct.

**Pull request merged**